.editorconfig

```diff
@@ -1,12 +1,16 @@
-# Defaults for all kinds of source code
+root = true
+
 [*]
-charset = utf-8
-end_of_line = lf
-indent_size = 4
 indent_style = space
-insert_final_newline = true
+indent_size = 4
+end_of_line = lf
+charset = utf-8
 trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+# For isort
+profile = black
```

Comment:
According to the `isort` docs, [`.editorconfig` is supported](https://pycqa.github.io/isort/docs/configuration/config_files.html#editorconfig) but [`pyproject.toml` is preferred](https://pycqa.github.io/isort/docs/configuration/config_files.html#editorconfig).

I've replaced explicit `isort` settings by the profile for Black in `pyproject.toml` in #756.

Reply:
Yes, this PR doesn't follow the most pure approach on config files for now, but the most practical.

Precommix supports non-python projects, where a `pyproject.toml` is not necessary. So it uses other dotfiles instead if possible.

I hope the world of linters gravitates around https://dot-config.github.io/ soon and then these problems start disappearing.